### PR TITLE
bump to 4.2.0

### PR DESCRIPTION
**Pull request by parkhomenko**
### Notes
bump to 4.2.0 release

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,8 @@
+## 4.2.0
+
+- Implement inactivity timeout for HTTP2 session
+- Cloudflare navigator object fix
```
Comment (faunaee) [inline CHANGELOG.md:4]:
```suggestion
- Implement a fix for the lack of a `navigator` object in Cloudflare Workers.
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,8 @@
+## 4.2.0
+
+- Implement inactivity timeout for HTTP2 session
```
Comment (faunaee) [inline CHANGELOG.md:3]:
```suggestion
- Improve HTTP2 session timeout handling.
- Add the `FAUNADB_HTTP2_SESSION_IDLE_TIME` environment variable
  to specify the default HTTP2 session timeout period.
```

**Review by faunaee — CHANGES_REQUESTED** [review]
Just a couple of suggestions to make the CHANGELOG appropriately specific. Otherwise, looks good!

File: CHANGELOG.md (hunk 1 above)
Comment (parkhomenko) [reply CHANGELOG.md:4]: thanks, I'll change it

File: CHANGELOG.md (hunk 2 above)
Comment (parkhomenko) [reply CHANGELOG.md:3]: thanks for corrections 👍

**Review by n400 — APPROVED** [review]

**Review by tigger9flow — APPROVED** [review]

**Review by faunaee — APPROVED** [review]
LGTM! 👍

**Pull request merged**